src/demo/models/guide.js

```diff
@@ -1,101 +1,108 @@
 import {getState, setState} from '../state';
 import {AppMode, Modes} from '../constants';
 import crabImage from '../../../public/images/seaCreatures/Crab.png';
+import turtleImage from '../../../public/images/seaCreatures/Turtle.png';
+import trash6PackImage from '../../../public/images/trash/Trash_6pack.png';
+import trashBottleImage from '../../../public/images/trash/Trash_Bottle.png';
+import trashCanImage from '../../../public/images/trash/Trash_Can.png';
+import guideFish1 from '../../../public/images/fish/guideFish/guideFish1.png';
 
 const guides = [
   {
     id: 'fishvtrash-training-init',
-    text: "Garbage dumped in the ocean and rivers affects water health and marine life.  In this activity you will program an artificial intelligence (AI) to identify trash.  Let's clean up the ocean!",
+    text: `Garbage dumped in the ocean and rivers affects water health and marine life.  In this activity you will "program" or "train" an artificial intelligence (AI) to identify fish or trash.  Let's clean up the ocean!`,
     when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
     style: 'Center',
     arrow: 'none',
-    image: crabImage
+    image: guideFish1
   },
   {
     id: 'fishvtrash-training-init2',
-    text: "Let's meet A.I.",
+    text: `Let's meet A.I.`,
     when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
     style: 'TopRight',
   },
   {
     id: 'fishvtrash-training-init3',
-    text: 'A.I. does not know if an object is a fish or trash, but it can process different images and identify patterns.',
+    text: `A.I. is an artifical inteligence and does not know if an object is a fish or trash, but it can process different images and identify patterns.`,
     when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
-    style: 'BottomRightCenter',
+    style: 'Center',
     arrow: 'none'
   },
   {
     id: 'fishvtrash-training-init4',
-    text: "To program A.I., use the buttons to label an image as either 'fish' or 'not fish'. This will train A.I. to do it on its own. Let's get started!",
+    text: `To program A.I., use the buttons to label an image as either "fish" or "not fish".  This will teach A.I. to do it on its own.  Let's get started!`,
     when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
     style: 'BottomMiddle',
   },
   {
     id: 'fishvtrash-training-pause1',
-    text: "Did you know 1 in 3 people do not have access to clean water?  Keep training to help A.I. identify trash in the water!",
+    text: `Did you know an estimated 17 billion pounds of plastic enters into the ocean from land-based sources each year?  Keep training to help A.I. identify what trash looks like.`,
```

Comment:
should be: "enters the ocean", no "into"

Reply:
fixed!